lib/Mojo/IRC.pm

```diff
@@ -345,6 +341,123 @@ sub ctcp {
   ":\001${_}\001";
 }
 
+=head2 channels
+
+  $self = $self->channels(sub { my ($self, $err, $channels) = @_; });
+
+Will retrieve available channels on the IRC server. C<$channels> has this
+structure on success:
+
+  {
+    "#convos" => {n_users => 4, topic => "[+nt] some cool topic"},
+  }
```

Comment:
I'm not sure. Parsing ```:hybrid8.debian.local 322 test10409 #convos 4 :[+nt] some cool topic``` gives me ```{params=>['test10409', '#convos', '4', '[+nt] some cool topic']}```. Should I assume that the mode can be removed from the topic?